juniper_official/Solutions/EX-Platform/IntStatsTable.yml

```diff
@@ -0,0 +1,35 @@
+---
+IntStatsTable:
+    rpc: get-interface-information
+    args:
+        extensive: True
+    item: physical-interface
+    key: name
+    view: IntStatsView
+
+IntStatsView:
+    groups:
+        ethernet: ethernet-autonegotiation/local-info
+    fields:
+        description: description
+        admin-status: admin-status
+        oper-status: oper-status
+        bpdu-error: bpdu-error
```

Comment:
bandwidth rule which is in progress by bindu requires speed. please add speed i.e... 

speed: speed